projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -93,7 +93,8 @@ The file contains two sections, [complex symbols](#complexSymbols) and [symbols]
 
 #### Translating this file {#TranslatingSymbolsFile}
```

Comment:
I'd move this paragraph at the end. Indeed the sentence before announces the two next paragraphs.